packages/person/package.json

```diff
@@ -37,7 +40,8 @@
     "url": "https://github.com/equinor/fusion-react-components/issues"
   },
   "dependencies": {
+    "@equinor/fusion-wc-person": "^1.2.1",
     "@equinor/fusion-react-button": "^0.8.3",
-    "@equinor/fusion-wc-person": "^0.6.1"
+    "@equinor/fusion-react-utils": "^2.0.5"
   }
-}
+}
```

Comment:
Please insert newline at end of file. Should be a setting in your IDE.

Reply:
can't, I have pretty on doc save